src/components/Icon/Add.tsx

```diff
@@ -0,0 +1,27 @@
+import * as React from "react";
+import { SVGProps } from "react";
+
+const SvgAdd = (props: SVGProps<SVGSVGElement>) => (
+  <svg
+    width={24}
+    height={24}
```

Comment:
I was wondering if we should have these as a `size` prop

Reply:
Width and height, is part of the SVGProps, it should override these values.

Reply:
What I meant was more to lock those values with a single `size` prop, rather than have `width` and `height` separately. As in, we don't want to allow people to set different `width` and `height`.

```suggestion
    width={size}
    height={size}
```

Reply:
Wondering about this comment?

Reply:
Addressed with https://github.com/Major-Eth/yearn-web-lib/pull/21/commits/8ac5a07a706c56f2bc95084c5a9e4d03db20e4fe

^ like that @phillip055